django/applications/catmaid/state.py

```diff
@@ -98,19 +99,20 @@ def make_all_links_query(link_ids, node_id, is_connector=False):
     else:
         return StateCheck(template % ("", "%s", ""), [node_id])
 
-def list_to_table(l, n=1):
-    args = None
```

Comment:
This set of changes let us say that args is never none

Reply:
:+1: